wire-library/wire-runtime/src/commonMain/kotlin/com/squareup/wire/internal/Internal.kt

```diff
@@ -148,3 +148,15 @@ fun countNonNull(a: Any?, b: Any?, c: Any?, d: Any?, vararg rest: Any?): Int {
   }
   return result
 }
+
+private val ESCAPED_CHARS = """([,\\\[\]{}])""".toRegex()
+
+/** Return a string where `,\]\{}` are escaped with a `\`. */
```

Comment:
Escaped chars list in the doc is inconsistent with `ESCAPED_CHARS`.

Reply:
Good catch

Reply:
fixed